modules/labs-react/core/stories/stories.tsx

```diff
@@ -3,7 +3,7 @@
 import {jsx} from '@emotion/core';
 import styled from '@emotion/styled';
 import withReadme from 'storybook-readme/with-readme';
-import {spacing} from '@workday/canvas-kit-react/core';
+import {spaceNumbers} from '@workday/canvas-kit-react/core';
 import type, {space} from '..';
```

Comment:
This was an interesting case that I had to manually update. Because the file also imported `space`. Because the codemod change would have created re-declaration issue, the codemod didn't modify it and threw an error.

I think we should eventually get rid of `space` in labs-core, but for now, I switched it to `spaceNumbers` manually to move ahead.